RELEASE.md

```diff
@@ -52,6 +52,19 @@ If a bug fix got accidentally merged into master after non-bug-fix changes in ma
 
 Maintaining the release branches for older minor releases happens on a best effort basis.
 
+### Updating the dependencies
+
+A few days before a major or minor release, update the dependencies:
```

Comment:
Perhaps use a more cautious language? E.g. "A few days before a major or minor release, consider updating the dependencies:" And then later add a warning to look out for any weirdness that might have happened. We should avoid the impression that a dependency update is always good and always "just works".

Reply:
Thank you for the review. I have added such a text now.

Reply:
Updating dependencies isn't always safe, but I think we should always be doing it before a release. Otherwise when it eventually does happen it'll be harder to deal with any resulting issues.

Reply:
I would propose to rephrase that if some dependencies can't be updated with a reasonable amount of efforts, an issue must be created for followup?

Reply:
That's reasonable.